Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromDDD.cc

```diff
@@ -34,243 +33,112 @@ GEMGeometryBuilderFromDDD::~GEMGeometryBuilderFromDDD()
 
 GEMGeometry* GEMGeometryBuilderFromDDD::build(const DDCompactView* cview, const MuonDDDConstants& muonConstants)
 {
-  std::string attribute = "ReadOutName"; // could come from .orcarc
-  std::string value     = "MuonGEMHits";    // could come from .orcarc
-  DDValue val(attribute, value, 0.0);
+  std::string attribute = "MuStructure"; // "ReadOutName"; // could come from .orcarc
```

Comment:
@ianna - I change this to read from Geometry/GEMGeometryBuilder/data/GEMSpecsFilter.xml, so we can loop through chambers and then loop through the associated eta partitions.
The SensitiveDetector eta partitions are the same as the previous version (GHA10X and GHA20X).